docs/src/man/basics.md

```diff
@@ -1510,32 +1510,97 @@ julia> german[Not(5), r"S"]
 
 ## Basic Usage of Transformation Functions
```

Comment:
I am pretty much done updating this section. I will work on the Broadcasting with Transformation Functions section next when I find some time.